x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/security/profile/ProfileCancellationIntegTests.java

```diff
@@ -166,11 +171,21 @@ private void disableSearchBlock() {
         }
     }
 
+    private boolean isShardSearchBlocked() {
+        for (PluginsService pluginsService : internalCluster().getInstances(PluginsService.class)) {
+            if (pluginsService.filterPlugins(SearchBlockPlugin.class).stream().anyMatch(SearchBlockPlugin::isShardSearchBlocked)) {
+                return true;
```

Comment:
Since the profile index only has a single primary shard. The shardSearch task should only be handled on a single node (either the one hosting the primary or the one hosting the replica). So it is not possible for it to be blocked on all nodes. In fact, the test fails (timeout) if we try to assert all nodes are blocked.